src/app/(website)/_components/RalatedBlog/RalatedBlog.tsx

```
@@ -1,116 +1,288 @@
+                    alt={post.heading || "Blog Image"}
+                    width={888}
+                    height={552}
+                    className="w-full h-[455px] object-cover rounded-t-md"
+                    priority
+                  />
+                </div>
+                <Link
+                  href={`/${post.category_id}/${post.subcategory_id}/${post.id}`}
+                >
+                  <p
+                    dangerouslySetInnerHTML={{ __html: post.heading }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)